scripts/format.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Are these scripts written from scratch or copied and modified? If they were copied, it would be useful to include in the comment where it was copied from.

Hopefully in the long term we can move back to d2-style, but I understand the need to do this now to be able to move forward.